docs/api-reference/next/image.md

```diff
@@ -95,7 +95,6 @@ The layout behavior of the image as the viewport changes size.
   - Ensure the parent element has `position: relative` in their stylesheet.
 - When `raw`[\*](#experimental-raw-layout-mode), the image will be rendered as a single image element with no wrappers, sizers or other responsive behavior.
   - If your image styling will change the size of a `raw` image, you should include the `sizes` property for proper image serving. Otherwise your image will receive a fixed height and width.
```

Comment:
I think this sentence needs be to adjusted?

Reply:
Maybe this sentence is a little awkward, but I think it's basically still correct. If you don't include sizes, the srcset works like a fixed image (1x and 2x only). I think "receive" is the confusing word here. I'll see if I can come up with something else.

Reply:
Changed it to: `If your image styling will change the size of a `raw` image, you should include the `sizes` property for proper image serving. Otherwise your image will be requested as though it has fixed height and width.`